docker-compose.yml

```diff
@@ -18,6 +18,9 @@ services:
       - type: volume
         source: ac-database
         target: /var/lib/mysql
+      - type: bind
+        source: ./data
+        target: /azerothcore/data
```

Comment:
the database docker is a multistage container. This bind will only get added to the last stage of the docker, which will not work as the /data is only used in the stage and the first time the docker is build and will not generate the databases correctly(it will create the databases but they will be empty). We should leave the `COPY data /azerothcore/data` command in the dockerfile

Reply:
this is a problem because the core build does not require those data, and this is a 4GB+ folder. the problem comes from the fact the same .dockerignore file is used for the 3 images. the build of acore-worldserver does not require ./data

Reply:
this is a problem because the core build does not require those data, and this is a 4GB+ folder. the problem comes from the fact the same .dockerignore file is used for the 3 images. the build of acore-worldserver does not require ./data

Reply:
Are you storing your map data in `./data` directory (the data folder should not be 4GB+)? When using docker you should actually be putting the map data into `docker/worldserver/data/`.(see https://www.azerothcore.org/wiki/Install-with-Docker)  This avoids the issue have having the map data get sent during the docker build context. Without the map data the `./data` folder is about ~462MB.